### ISSUE TITLE
Two errors

### ISSUE THREAD

**Issue by ocelot1300** [mezbaul-h/june#15]
Hi! I get two errors when running this:

FutureWarning: The input name `inputs` is deprecated. Please make sure to use `input_features` instead.
  warnings.warn(
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

**Comment by mezbaul-h** [comment]
Could you paste the full verbose log?

**Comment by ocelot1300** [comment]
C:\Users\***>june-va -v
[system]> LLM model initialized (model_id=llama3.1:8b-instruct-q4_0; device=cpu)
[system]> No system prompt provided.
[system]> STT model initialized (model_id=openai/whisper-small.en; device=cpu)
[system]> TTS model initialized (model_id=tts_models/en/ljspeech/glow-tts; device=cpu)
[system]> Listening for sound...
[system]> Sound detected, starting recording...
[system]> Silence detected, stopping recording...
[system]> Transcribing audio...
C:\Users\***\AppData\Local\Programs\Python\Python310\lib\site-packages\transformers\models\whisper\generation_whisper.py:509: FutureWarning: The input name `inputs` is deprecated. Please make sure to use `input_features` instead.
  warnings.warn(
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
[user]> Hello there.
[assistant]> It's nice to meet you. Is there something I can help you with, or would you like to chat?

**Comment by mezbaul-h** [comment]
that's a warning, not an error. the warning is triggered by `transformers` which is integrated into my STT model. https://github.com/huggingface/transformers/issues/33498 for ref. once they fix and release a new version, it will be gone. regardless, everything should work, for the most part.
